DESCRIPTION

```diff
@@ -25,6 +25,7 @@ Imports:
     fs,
     ggplot2,
     glue,
+    here,
```

Comment:
Not sure why `skip_on_ci()` and `skip_if_not_installed("here")` did not do the trick, and the checks were still expecting `here::here()` to be imported. For now I just added the dependency. Before a CRAN release I'd like to try prune as many packages as possible anyway.